docs/content/docs/development/write-table.md

```diff
@@ -191,7 +191,8 @@ The following parameters determine when to stop writing:
 
 There are three main places in the Table Store's sink writer that take up memory:
 - MemTable's write buffer, shared and preempted by all writers of a single task.
-  This memory value can be adjusted by the `write-buffer-size` option.
+  This memory value can be adjusted by the `write-buffer-size` option. You need
+  to ensure that the number of pages is at least greater than the number of writers.
```

Comment:
```suggestion
  to ensure that the number of pages(`write-buffer-size`/`page-size`) is at least greater than the number of writers(the parallelism of the execution environment).
```

Reply:
I think I can delete this document, it is difficult to explain clearly the description of this writers, and this behavior we have to fix it later.